### DISCUSSION TITLE
posts/music/2024/12/11/01/12%EC%9B%94-11%EC%9D%BC-%EB%8C%80%EC%B6%A9-%EB%A7%8C%EB%93%A0%EA%B1%B0/

### DISCUSSION THREAD

**Discussion by giscus** [melted-mint/fuwari-edited#7] [Comments]
# posts/music/2024/12/11/01/12%EC%9B%94-11%EC%9D%BC-%EB%8C%80%EC%B6%A9-%EB%A7%8C%EB%93%A0%EA%B1%B0/

Sample이에요.

https://meltedmint.com/posts/music/2024/12/11/01/12%EC%9B%94-11%EC%9D%BC-%EB%8C%80%EC%B6%A9-%EB%A7%8C%EB%93%A0%EA%B1%B0/

**Comment by applemint35** [comment]
밝고 리듬감 있어서 좋은거같네요
약간 옛날 포켓몬같은 게임 마을 bgm 느낌나는거 같아요

**Reply by meltedmint** [reply]
그런가?
으으으으음...
나쁘지는 않은거 같기도요 에
